jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -26,6 +27,17 @@
 from jax import numpy as jnp
 
 class JaxToTfTestCase(jtu.JaxTestCase):
+  @classmethod
+  def setUpClass(cls):
+    # Initialize categorizer
+    categorizer = jax2tf.jax2tf.Categorizer()
+    # Register limitation summary atexit printing handler
+    atexit.register(categorizer.pprint_limitations, None)
+    # Monkey-patch jax2tf.TensorFlowTrace.get_primitive_impl to wrap the
+    # resulting primitive in the categorizer.
+    original_impl = jax2tf.jax2tf.TensorFlowTrace.get_primitive_impl
```

Comment:
I think it would be better to restore the `original_impl` on exit from this class.

Reply:
Right, good catch!